sensor.js

```diff
@@ -51,11 +51,15 @@ rtl433Plugin.prototype = {
 
 function rtl433Server() {
   // console.log("This", this);
+  console.log("This", this);
   var childProcess = require('child_process');
   var readline = require('readline');
   var previousMessage;
   this.log("Spawning rtl_433");
-  var proc = childProcess.spawn('pkill rtl_433;/usr/local/bin/rtl_433', ['-q', '-F', 'json', '-C', 'si'], {
+//  var proc = childProcess.spawn('pkill rtl_433;/usr/local/bin/rtl_433', ['-q', '-F', 'json', '-C', 'si'], {
+  // DO NOT RUN homebridge AS ROOT
+  // start outside homebride (as root) the cmd: rtl_433 -v -F json -C si -M protocol > /tmp/rtl433.json
+  var proc = childProcess.spawn('/usr/bin/truncate -s 0 /tmp/rtl433.json;/usr/bin/tail', ['-F','/tmp/rtl433.json'], {
```

Comment:
Can we make this alternative spawn command a configuration option, and have the existing spawn command be the default.  This way it is not a breaking change for existing users.

Not sure what to to call the config option, but something simple like rootless

Also this approach of a simple tail will be problematic for people, once the tail file size exceeds the file system size.

Can you also update the README with the details of new option and how to set it up.

PS If google 'ubuntu rtl-sdr udev rules' it will show several examples of configuring udev to all rootless access to the rtf_sdr

Reply:
You are 100% right, i will change the code to avoid the breaking change.
And may be also add support for other accessories like power meter.
// {"time" : "2022-03-18 00:46:31.387218", "protocol" : 12, "model" : "Oregon-CM180", "id" : 44336, "battery_ok" : 1, "power_W" : 982, "energy_kWh" : 218.695 }